web/src/lib/components/assets/thumbnail/image-thumbnail.svelte

```diff
@@ -1,7 +1,7 @@
 <script lang="ts">
-	import { onMount } from 'svelte';
-	import { lazyLoad } from 'unlazy';
```

Comment:
Can we remove this dependency from package.json now?

Reply:
Guess so, let me check